Resources/config/dbal.xml

```diff
@@ -23,11 +23,7 @@
         <service id="Doctrine\DBAL\Driver\Connection" alias="database_connection" public="false" />
         <service id="Doctrine\DBAL\Connection" alias="database_connection" public="false" />
 
-        <service id="doctrine.dbal.logger.chain" class="%doctrine.dbal.logger.chain.class%" public="false" abstract="true">
-            <call method="addLogger">
-                <argument type="service" id="doctrine.dbal.logger" />
```

Comment:
wasn't this call needed ? I don't see its replacement

Reply:
The replacement is in PHP, it's the very first lines of this PR